### PR TITLE
Add OrangePI PC+ target

### PR DESCRIPTION
**Pull request by volodymyr-ostrovskyi**
Based on Orange Pi Plus 2E configuration with some modifications.
Audio options have been changed.
Includes fix wifi.

### CODE REVIEW

File: opi_pc_plus/opi_pc_plus.mk [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+PRODUCT_MANUFACTURER := xunlong
+PRODUCT_HAS_EMMC := true
+
+UBOOT_DEFCONFIG := orangepi_pc_plus_defconfig
+KERNEL_SRC       := kernel/glodroid-stable
+KERNEL_DEFCONFIG := $(KERNEL_SRC)/arch/arm/configs/sunxi_defconfig
+KERNEL_FRAGMENTS += \
+    device/glodroid/platform/common/sunxi/sunxi-common.config \
+
+KERNEL_DTB_FILE := sun8i-h3-orangepi-plus2e.dtb
```
Comment (rsglobal) [inline opi_pc_plus/opi_pc_plus.mk]: Hi, did you meant `pc-plus`, instead of `plus2e`

File: opi_pc_plus/opi_pc_plus.mk [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2020 Roman Stratiienko (r.stratiienko@gmail.com)
```
Comment (rsglobal) [inline opi_pc_plus/opi_pc_plus.mk]: Please change all such lines to:
Copyright (C) 2021 GloDroid project

Comment (volodymyr-ostrovskyi) [reply opi_pc_plus/opi_pc_plus.mk]: Done.

File: opi_pc_plus/audio.opi_pc_plus.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<audiohal>
+    <mixer card="0">
+        <init>
+            <ctl name="Line Out Playback Volume" val="31" />
+            <ctl name="Line Out Playback Switch" val="1"/>
+            <ctl name="DAC Playback Switch" val="1"/>
+            <ctl name="Mic1 Capture Switch" val="1"/>
+            <ctl name="Mic2 Capture Switch" val="1"/>
```
Comment (rsglobal) [inline opi_pc_plus/audio.opi_pc_plus.xml]: Let's just keep one ob-board mic for now.

**Comment by rsglobal** [comment]
@volodymyr-ostrovskyi ,

Please add your Signed-off-by line to the commit message.

**Comment by rsglobal** [comment]
Thanks, nerging

**Pull request merged**